equipmanager.lic

```diff
@@ -212,7 +212,7 @@ class EquipmentManager
 
     todo.all? do |held_item|
       if info = worn_items.find { |item| item.short_regex =~ held_item }
-        stow_helper("wear my #{info.short_name}", info.short_name, 'You sling', 'You attach', 'You .* unload', 'You strap', 'You slide', 'You work your way into', 'You spin', '^You ', 'You carefully loop')
+        stow_helper("wear my #{info.short_name}", info.short_name, 'You sling', 'You attach', 'You .* unload', 'You strap', 'You slide', 'You work your way into', 'You spin', '^You ', 'You carefully loop', 'slide effortlessly onto your')
```

Comment:
Yeah, still locks up. Let me test more tonight and I'll see. It was checking something shortened. Forget the exact, but I'll get it tested tonight